### PR TITLE
Update index.rst to announce release of PyGMT 0.5.0

### PR DESCRIPTION
**Pull request by michaelgrund**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -174,14 +174,14 @@
             <div class="col-sm-4 front-block-text">
                <h2>News</h2>
 
-               <ul>
+               <ul>
```
Comment (weiji14) [inline index.rst:177]:
```suggestion
               <ul>            
```

File: index.rst [hunk 2]
```diff
@@ -174,14 +174,14 @@
             <div class="col-sm-4 front-block-text">
                <h2>News</h2>
 
-               <ul>
+               <ul>            
                <li>
-                  2021-09-15:
-                  GMT now has an Instagram handle <a href="https://www.instagram.com/genericmappingtools/">@genericmappingtools</a>!
+                  2021-10-30:
```
Comment (weiji14) [inline index.rst:179]: Maybe better to stick with the release date on the changelog.
```suggestion
                  2021-10-29:
```

**Review by weiji14 — COMMENTED** [review]
Thanks @michaelgrund! Just two tiny suggestions.

File: index.rst (hunk 2 above)
Comment (michaelgrund) [reply index.rst:179]: Good point! Thanks @weiji14 😉

File: index.rst (hunk 1 above)
Comment (michaelgrund) [reply index.rst:177]: How to commit this suggestion @weiji14 ?

Comment (weiji14) [reply index.rst:177]: Oops, forgot to delete the blankspace. Should be ok now.

**Review by weiji14 — APPROVED** [review]

**Comment by weiji14** [comment]
I don't have write access to merge this PR. @seisman, could you help?

**Review by seisman — APPROVED** [review]

**Pull request merged**